added preconfigured storage buses

boubou19 opened the pull request:
closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17821
requires https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/615 and https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/615

Add recipes for preconfigured storage and fluid storage buses, from priority 1 to priority 24.

![image](https://github.com/user-attachments/assets/4e625e00-cfdd-4e72-bfba-f12c121d8e18)
(tooltips added with CustomTooltips)

serenibyss commented:
This causes some recipe conflicts, particularly the bottom one:
![image](https://github.com/user-attachments/assets/568446d5-124d-4a7b-8df8-be4f55c91a99)

boubou19 commented:
added the shapeless crafting recipes:
![image](https://github.com/user-attachments/assets/26fd4e3b-4edd-4321-bad4-ac41cf973347)
![image](https://github.com/user-attachments/assets/743762cd-b4ca-41fb-8623-2a6d76a905b4)

and i moved recipes from assembler to circuitassembler to avoid any conflict:
![image](https://github.com/user-attachments/assets/8e4dfa00-dc7b-485a-9252-fbfb8887063e)
![image](https://github.com/user-attachments/assets/b3b19685-7cbb-4152-8afb-373607960152)
